ci-operator/step-registry/upi/deprovision/gcp/upi-deprovision-gcp-commands.sh

```diff
@@ -8,6 +8,12 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 
 export HOME=/tmp
 
+if [[ -s "${SHARED_DIR}/xpn.json" ]] && [[ -f "${CLUSTER_PROFILE_DIR}/xpn_creds.json" ]]; then
+  echo "Activating XPN service-account..."
+  GOOGLE_CLOUD_XPN_KEYFILE_JSON="${CLUSTER_PROFILE_DIR}/xpn_creds.json"
+  gcloud auth activate-service-account --key-file="${GOOGLE_CLOUD_XPN_KEYFILE_JSON}"
+  GOOGLE_CLOUD_XPN_SA=$(jq -r .client_email "${GOOGLE_CLOUD_XPN_KEYFILE_JSON}")
+fi
```

Comment:
We are activating this service account here, and then we are activating another a few lines later. This first activation is wrapped in an `if`. Do we want to activate both, or should the second activation be wrapped in an `else` so we are only activating one per run?

Reply:
The line 14 activating is to conditionally activate the XPN service-account, which would enable accessing the XPN project resources with the "--project XXX" & "--account YYY" options. The line 18 & 19 would then activate the service-account of the normal/service project which is taken as the target project where to deploy OCP cluster. That's it, thanks!